src/vim.h

```diff
@@ -435,6 +435,15 @@ typedef unsigned short sattr_T;
  */
 typedef unsigned int u8char_T;	// int is 32 bits or more
 
+/*
+ * The vimlong_T has sizeof(vimlong_T) >= 2 * sizeof(int).
+ * One use is simple handling of overflow in int calculations.
+ */
+#ifdef HAVE_NO_LONG_LONG
```

Comment:
```suggestion
#if defined(VMS) && defined(VAX)
```

Using system definitions seems more consistent with what's already in `vim.h`

Then can get rid of definitions and references to `HAVE_NO_LONG_LONG`

Reply:
Introducing HAVE_NO_LONG_LONG was Bram's idea back in the time as there are other different platforms and architectures that do not have long long time at all.
Now searching the code, seems you are right. Just VAX is the only one left using it.

I would prefer leaving HAVE_NO_LONG_LONG as this clearly explains why we are doing this acrobatics with the types (also it is defined in the os_*.h files that clearly narrows the scope)